### PR TITLE
Add prop types for visualizer component

### PR DESCRIPTION
**Pull request by phanlyhuynh**

### CODE REVIEW

**Comment by aladhims** [comment]
Please use define type explicitly for each field in the deadline @phanlyhuynh

File: src/visualizer/deadlineVisualizer.js [hunk 1]
```diff
@@ -96,4 +97,8 @@ const DeadlineVisualizer = ({deadline}) => {
     );
 }
 
-export default DeadlineVisualizer;
\ No newline at end of file
+DeadlineVisualizer.propTypes = {
+    deadline: PropTypes.shape().isRequired
```
Comment (aladhims) [inline src/visualizer/deadlineVisualizer.js:101]: Please use define type explicitly for each field in the deadline @phanlyhuynh

**Review by aladhims — CHANGES_REQUESTED** [review]
Please check above comment

**Comment by phanlyhuynh** [comment]
@aladhims I fixed it. Please check for me? Thanks :D

**Review by aladhims — APPROVED** [review]
LGTM 👍

**Pull request merged**